lib/group_site_manager.py

```diff
@@ -312,7 +312,10 @@ def _upload_bfmr(self, numbers) -> None:
 
   def _upload_bfmr_batch(self, numbers) -> None:
     group_config = self.config['groups']['bfmr']
+    former_headless = self.driver_creator.args.no_headless
```

Comment:
what's up with this?

Reply:
It's the same as what's already going on with Melul uploads.  We don't have a good programmatic way to actually set this, so overriding args (and setting them back) is the best we have for now.

We really do need to rewrite how driver_creator works so that we can specify on each new() call if we want headless or not.

Reply:
No, I mean why do we need to run BFMR headless? We haven't had to in the past.

Reply:
Dunno, this started being necessary as of Chrome v80.  It wasn't working without it though.